netstandard/tools/RemoveDepsFileConflicts.cs

```diff
@@ -0,0 +1,157 @@
+﻿// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
+using Microsoft.Build.Framework;
+using Microsoft.Build.Utilities;
+using Microsoft.Extensions.DependencyModel;
+using System;
+using System.Collections.Generic;
+using System.IO;
+using System.Linq;
+
+namespace Microsoft.DotNet.Build.Tasks
+{
+    public partial class RemoveDepsFileConflicts : Task
+    {
+        Dictionary<string, HashSet<string>> compileConflicts = new Dictionary<string, HashSet<string>>(StringComparer.OrdinalIgnoreCase);
+        Dictionary<string, HashSet<string>> runtimeConflicts = new Dictionary<string, HashSet<string>>(StringComparer.OrdinalIgnoreCase);
+
+        [Required]
+        public string DepsFilePath { get; set; }
+
+        [Required]
+        public ITaskItem[] Conflicts { get; set; }
+
+        public override bool Execute()
+        {
+            LoadConflicts();
+
+            DependencyContext sourceDeps;
+            using (var sourceStream = File.Open(DepsFilePath, FileMode.Open, FileAccess.Read, FileShare.Read | FileShare.Delete))
```

Comment:
Convenience function has the wrong defaults.  Best practice is to specify delete sharing so that folks can move the file out of the way and replace it.  Not passing delete sharing is one of the primary causes behind file in use situations.  We can't change the defaults due to compat.  Probably need a new convenience function.